podman.md

```diff
@@ -0,0 +1,123 @@
+# SlicOps and Podman
+
+SliCops is deployed in a Podman container image.
+
+## Building the image
+
+From the repository root run:
+
+```
+curl radia.run | container-build
```

Comment:
curl https://radia.run | bash -s container-build

Reply:
We may want podman.md to be somewhere else, e.g. the wiki. I think that's better having it here. Although you could refer to it in the README.md, which would be helpful.

Reply:
d1aa076